source/Calamari.Tests/KubernetesFixtures/Terraform/Clusters/providers.tf

```diff
@@ -2,7 +2,7 @@ terraform {
   required_providers {
     azurerm = {
       source  = "hashicorp/azurerm"
-      version = ">= 2.63.0"

```

Comment:
This looks like it has crept in from another commit? Probably all good to drag it forward if the tests are green

Reply:
Yeah, that was added in the last PR to fix the failing Kubernetes tests 😄